### DISCUSSION TITLE
Tips on using multiple duplicate instances of Budibase?

### DISCUSSION THREAD

**Discussion by Nasnl** [Budibase/budibase#5971] [Help] [answered]
I'm planning to set up a Budibase app that I want to duplicate several times across the organisation so it can be used in the same way by various departments, but each with their own data. I guess the way to go is using several containers and configure each individually.

Does anyone have experience with this approach and have tips, do's or don'ts?

Thanks!

**Comment by melohagan** [comment]
Hi @Nasnl 

So in this case all of the departments would be using the same app? Might be a little hassle exporting and importing the app every time it's updated to each container, but could be done.  

In my opinion I would avoid the internal DB, and use an external DB (not containerised) as well. You never know if you might want to join or otherwise collate data between the departments, and if you're using the internal DB then it's going to be more awkward to do that.
Probably depends on how scalable it needs to be.

**Reply by Nasnl** [reply]
@melohagan Tx for your feedback. 

I recognize the maintenance challenge with updates. I'm looking into automation solutions for that (but not much so far).

The external shared database is probably the best way forward indeed, working with dept-prefixes in the table names, probably. I realize it'll only scale up to a certain level...

**Reply by csennewald** [reply]
Did you get it done?
I encountered several problems with the Couch DB.
Only changing the ports was not the solution :(

**Reply by mike12345567** [reply, marked as answer]
> Did you get it done? I encountered several problems with the Couch DB. Only changing the ports was not the solution :(

Hi @csennewald - it is not possible to host multiple different installations of Budibase on the same CouchDB node. There is a multi-tenancy mode which allows Budibase to operate multiple instances from one installation but support for this would only be available on the enterprise tier.
